apps/test/unit/lib/kits/maker/boards/circuitPlayground/PlaygroundComponentsTest.js

```diff
@@ -41,20 +44,28 @@ describe('Circuit Playground Components', () => {
     // Our sensors and thermometer block initialization until they receive data
     // over the wire.  That's not great for unit tests, so here we stub waiting
     // for data to resolve immediately.
-    sinon.stub(EventEmitter.prototype, 'once');
-    EventEmitter.prototype.once
+    sinon.stub(five.Sensor.prototype, 'once');
```

Comment:
Re: stubbing, I was thinking something like this (if it doesn't work easily for some reason, I think fine to leave as-is)

```
function stubComponent(component) {
    // component would be a reference to five.Sensor, etc.
    sinon.stub(component.prototype, 'once');
    component.prototype.once
      .withArgs('data')
      .callsFake(function(_, callback) {
        // Pretend we got a real analog value back on the component's pin.
        setSensorAnalogValue(this, INITIAL_ANALOG_VALUE);
        callback();
      });
}

unstubComponent(component) {
  component.prototype.once.restore();
}
```

Reply:
Looks great!